src/BoundingBoxCameraSensor.cc

```diff
@@ -19,7 +19,9 @@
 
 #include <gz/msgs/image.pb.h>
```

Comment:
msgs::Boolean ?

Reply:
Good catch, I can't really detect the things that are already pulled in transitively.